tools/indexer/example/src/main.rs

```diff
@@ -215,11 +264,30 @@ async fn listen_blocks(mut stream: mpsc::Receiver<near_indexer::BlockResponse>)
 }
 
 fn main() {
-    let home_dir: Option<String> = env::args().nth(1);
-
+    openssl_probe::init_ssl_cert_env_vars();
```

Comment:
Let's add a comment that we use it to automatically search the for root certificates to perform HTTPS calls (sending telemetry and downloading genesis)